homeassistant/components/tuya/light.py

```diff
@@ -140,6 +140,8 @@ class TuyaLightEntityDescription(LightEntityDescription):
     "gyd": (
         TuyaLightEntityDescription(
             key=DPCode.SWITCH_LED,
+            name="floodlight",
+            icon="mdi:light-flood-down",
```

Comment:
This is not needed to set. 
Additionally, this device type isn't just used for floodlights. Please remove these 2 lines.

Reply:
Okay yes thanks, will remove these 2 lines.